dso-l2/src/main/java/com/tc/objectserver/impl/DistributedObjectServer.java

```diff
@@ -241,6 +242,7 @@
 import com.tc.objectserver.handler.ServerManagementHandler;
 import com.tc.operatorevent.TerracottaOperatorEvent;
 import com.tc.operatorevent.TerracottaOperatorEventCallback;
+
```

Comment:
Probably want to make sure that there aren't any gratuitous formatting changes, in the import list.